### ISSUE TITLE
React Hook useMemo has a missing dependency

### ISSUE THREAD

**Issue by glauberfc** [tamagui/tamagui#341]
Looks like the file `apps/next/pages/_app.tsx` has a missing `useMemo` dependency in `next-expo-solito` starter:

```
function MyApp({ Component, pageProps }: SolitoAppProps) {
  const [theme, setTheme] = useRootTheme()

  const contents = useMemo(() => {
    return <Component {...pageProps} />
  }, [pageProps])
```

Warning:
`React Hook useMemo has a missing dependency: 'Component'. Either include it or remove the dependency array.eslint[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)`

**Comment by natew** [comment]
Can you send a PR?

**Comment by natew** [comment]
Fixed in f39da1078
